src/ng/compile.js

```diff
@@ -212,10 +212,22 @@
  *
  *
  * #### `bindToController`
- * When an isolate scope is used for a component (see above), and `controllerAs` is used, `bindToController: true` will
+ * This property can be used to bind scope properties directly to the controller. It can be either
+ * `true` or an object hash with the same format as the `scope` property. Additionally,
+ * `controllerAs` must be used for it to work.
```

Comment:
This makes it sound like the `controllerAs` property must be used (which is not the case).
To avoid confusion, I would re-phrase this to make it clear that a controller alias must be specified (either using `controllerAs` or `controller: '... as ...'`).